### DISCUSSION TITLE
What are Dynamic content localization and Entities API?

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#8353] [General]
On the page [OrchardCore Features](https://docs.orchardcore.net/en/dev/docs/reference/modules/Features/), I've read that **Dynamic content localization** and **Entities API** are currently in development or expected for an upcoming version of Orchard Core. What are they? Where can I find any details on them?

**Comment by hishamco** [comment]
Basically dynamic localization are the process to localize non static content, something like content type, content fields names which are not be static and can be changed at any point of time, I started a PR #4466 long time ago to provide this functions - still in progress -  also @Skrypt trying to do some progress for his own purpose if I'm not wrong

If there anything else, feel free to ask

**Reply by DotCat1985** [reply]
Ok. Interesting. Thank you :)

**Comment by DotCat1985** [comment]
Anyone can tell me about Entities API?

**Reply by Skrypt** [reply]
No idea what it is. Maybe some work around Entities API, that would be YesSQL part of things.
